lib/Runtime/Library/JavascriptObject.cpp

```diff
@@ -93,6 +94,38 @@ Var JavascriptObject::EntryHasOwnProperty(RecyclableObject* function, CallInfo c
     JIT_HELPER_END(Object_HasOwnProperty);
 }
 
+Var JavascriptObject::EntryHasOwn(RecyclableObject* function, CallInfo callInfo, ...)
+{
+    JIT_HELPER_REENTRANT_HEADER(Object_HasOwn);
+    PROBE_STACK(function->GetScriptContext(), Js::Constants::MinStackDefault);
+
+    ARGUMENTS(args, callInfo);
+    ScriptContext* scriptContext = function->GetScriptContext();
+
+    Assert(!(callInfo.Flags & CallFlags_New));
+
+    RecyclableObject* dynamicObject = nullptr;
+    if (args.Info.Count < 2 || FALSE == JavascriptConversion::ToObject(args[1], scriptContext, &dynamicObject))
```

Comment:
Maybe a nit: this condition is a bit hard to rid. If understand it correctly we throw if there is less than 2 arguments or we can't convert `args[1]`  to an object - which I think would be the first user-accessible argument to `hasOwn`.

Reply:
This is correct but if it's unclear I could add a comment/make it neater